cmd/repository/default-pkg-registry_test.go

```diff
@@ -2,21 +2,13 @@ package repository
 
 import (
 	"fmt"
-	"os"
-	"path/filepath"
 	"testing"
-	"time"
 
 	"github.com/criteo/command-launcher/internal/command"
 	"github.com/stretchr/testify/assert"
 )
 
-func generateTestRegistryFile(path string, numOfPkgs int, numOfCmds int) (*defaultRegistry, error) {
-	reg, err := LoadRegistry(path)
-	if err != nil {
-		return nil, err
-	}
-
+func generateTestRegistryFile(reg Registry, numOfPkgs int, numOfCmds int) error {
```

Comment:
The default registry doesn't generate any file, we should not call it generateTestRegistryFile